### DISCUSSION TITLE
How can i use the current longhorn helmchart in main of this repo

### DISCUSSION THREAD

**Discussion by repudi8or** [longhorn/longhorn#3948] [General]
tl;dr https://charts.longhorn.io seems outdated/disconnected from https://github.com/longhorn/longhorn and i would like to see a "nightly" or "latest" build of a helm repo from the source of https://github.com/longhorn/longhorn

here is a copy/paste of an unanswered thread i posted in the longhorn room of the cncf Slack that provides details.

hi room.... i need to make use of the serviceAccount annotations described at https://github.com/longhorn/longhorn/blob/master/chart/templates/serviceaccount.yaml#L7 but this code is not in the latest release of the helm chart (v1.2.4) .... is there a helm repo'ish way for me to use the current charts on master of https://github.com/longhorn/longhorn  ???

this serviceAccount.annotations code is not in https://github.com/longhorn/charts/blob/master/charts/longhorn/templates/serviceaccount.yaml

this seems sort of weird to me when the commit for that functionality was on the 3rd of Jan - https://github.com/longhorn/longhorn/commit/1dedf7d37fb5b614f754b10f37f8e5f66eea00fe

and v1.2.4 was released on the 4th march (much later than the commit)

**Comment by repudi8or** [comment]
Longhorn just "released"  Longhorn v1.3.0-preview1 release but the helm repo has not been updated, meaning i can not deploy v1.3.0-preview1 using https://charts.longhorn.io

I really want to be able to deploy Longhorn v1.3.0-preview1 from helm

**Reply by innobead** [reply]
Right now, the charts repo is only for official releases only. 

We are thinking about this to have another repo for non-official releases, and it will probably be implemented in the future.

**Comment by repudi8or** [comment]
actually i just checked and the docs for v1.3.0-preview1 include this paragraph

"
Installation
Longhorn supports 3 installation ways including Rancher catalog, Kubectl, and Helm. Follow the installation instructions [here](https://v1-3-0--longhornio.netlify.app/docs/latest/deploy/install/)."

but this links to the standard longhorn helm install guide which describe using https://charts.longhorn.io/ which does NOT currently include v1.3.0-preview1.... 

so this "supports 3 installation ways" is not actually currently accurate in reference to v1.3.0-preview1

**Reply by joshimoo** [reply]
It will be true once v1.3.0 is released, since the documentation is for the upcoming v1.3.0 release.
If you want to try the preview for now you can either check it out locally and deploy it via helm from there.
Or much quicker just `kubectl apply -f https://raw.githubusercontent.com/longhorn/longhorn/master/deploy/longhorn.yaml` to just deploy the regular yaml.

**Since we don't support upgrades from preview releases, we recommend only using them for testing.**
